### ISSUE TITLE
Yarn installs incompatible version of datatables.net

### ISSUE THREAD

**Issue by chiatt** [archesproject/arches#10615]
Yarn installs an incompatible version of datatables.net since the datatables 2.0 release. In package.json datatables.net is listed as 
```
datatables.net "~1.12.1"
```
This should allow only patch level upgrades. However after running yarn install, datatables.net 2.0.0 is installed which has a different folder structure than 1.12.1. This results in the following error during the webpack build:

```
ERROR in ./media/node_modules/datatables.net-buttons/js/buttons.html5.min.js 
Module not found: Error: Can't resolve 'datatables.net' in 'media/node_modules/datatables.net-buttons/js'
```
Reported by @bferguso and reproduced by me

**Comment by chrabyrd** [comment]
A temporary fix for this is to remove the tilde range operator in `package.json`

**Comment by chrabyrd** [comment]
Alternatively, I'm not sure why we're keeping to 1.x versions and not upgrading to 2.x . If it's possible to upgrade we can reconfigure how `nodeModulesPaths` is configured.

**Comment by jacobtylerwalls** [comment]
Maybe 6.2.x and 7.5.x gets the tilde fix, but we upgrade 7.6 to yarn 2 since it's an LTS?

**Comment by chrabyrd** [comment]
Oh sorry I was unclear, I mean upgrade `datatables.net` to the 2.x versions
